src/OpenTelemetry/Metrics/Aggregators/Aggregator.cs

```diff
@@ -25,6 +27,14 @@ namespace OpenTelemetry.Metrics.Aggregators
     public abstract class Aggregator<T>
         where T : struct
     {
+        private long startTime;
+        private long checkpointStartTime;
+
+        protected Aggregator()
+        {
+            this.startTime = DateTime.UtcNow.ToBinary();
```

Comment:
Inststead of more expensive encoding of Kind into binary, you can use this:

```suggestion
            this.startTime = DateTimeOffset.UtcNow.Ticks;
```

(also use DateTimeoffset instead of DateTime may be good)

Reply:
👍 

Also it is worthy to clarify if the time is monotonic or not. This thread has some information https://github.com/dotnet/runtime/issues/15207.

Reply:
Ah, good suggestion. Thanks, done!

Not sure I'm totally following the suggestion here. The issue is still open. I see some suggestions to use `Stopwatch.GetTimestamp` which at least in some cases uses `DateTime.UtcNow`. I'd note here that we're already exposing the `MetricData.Timestamp` as a `DateTime`